sdk/keyvault/azure-keyvault-secrets-apis/azure/keyvault/secrets/client.pyi

```diff
@@ -0,0 +1,23 @@
+# Stubs for azure.keyvault.secrets.client (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ._shared import KeyVaultClientBase
+from .models import DeletedSecret, Secret, SecretProperties
+from azure.core.paging import ItemPaged
+from datetime import datetime
+from typing import Any, Optional
+
+class SecretClient(KeyVaultClientBase):
+    def get_secret(self, name: str, version: str=..., **kwargs: Any) -> Secret: ...
+    def set_secret(self, name: str, value: str, content_type: Optional[str]=..., not_before: Optional[datetime]=..., expires: Optional[datetime]=..., **kwargs: Any) -> Secret: ...
+    def update_secret_properties(self, name: str, version: Optional[str]=..., content_type: Optional[str]=..., not_before: Optional[datetime]=..., expires: Optional[datetime]=..., **kwargs: Any) -> SecretProperties: ...
+    def list_secrets(self, **kwargs: Any) -> ItemPaged[SecretProperties]: ...
```

Comment:
`list_secret_properties`? Or `list_secrets_properties`?

Reply:
@lmazuel had the same comment. We've struggled to find a good name for this (and its equivalents across Key Vault), so feedback here is most welcome. Here's some context for the status quo:

`list_secrets` is accurate and to the point in that the method returns a list of all secrets in the vault, and tells you *almost* everything you'd want to know about them. However, in line with Key Vault's permissions model, the list items don't contain secret values.

I'm averse to `list_*_properties` because to me it implies retrieving data that maybe I can't get anywhere else, and suggests `SecretProperties` is so important we name methods after it. But `SecretProperties` isn't a service concept, it's merely the name we applied to the object returned by `list_secrets`. It's everything `get_secret` gives you minus the secret value. A more precise, if unconventional, name might be `SecretWithoutValue`.

Reply:
`list_secrets_without_values` or `list_secrets_no_values` are not the worst names. Could be condensed maybe `list_abdridged_secrets`, `list_partial_secrets`

Reply:
The suggestion (loosely translated from the javascript and .NET/C# discussions on the same topic) is something like `list_properties_of_secrets`

Reply:
Agreed with list_properties_of_secrets

Reply:
#7797